app/uk/gov/hmrc/cataloguefrontend/createrepository/CreateRepositoryController.scala

```diff
@@ -114,12 +113,14 @@ class CreateRepositoryController @Inject()(
     ).async: request =>
       given Request[AnyContent] = request
       (for
-        userTeams   <- EitherT.pure[Future, Result](cleanseUserTeams(request.retrieval))
-        repoTypeOut <- getRepoTypeOut()
-        result      =  repoTypeOut.repoType match
-                         case RepoType.Prototype => Ok(createPrototypePage(CreatePrototype.form, userTeams.filterNot(_ == TeamName("Designers"))))
-                         case RepoType.Test      => Ok(createTestPage(CreateTest.form, userTeams))
-                         case RepoType.Service   => Ok(createServicePage(CreateService.form, userTeams))
+        userTeams         <- EitherT.pure[Future, Result](cleanseUserTeams(request.retrieval))
+        githubTeams       <- EitherT.liftF(teamsAndReposConnector.allTeams())
+        filteredUserTeams =  userTeams.filter(ut => githubTeams.exists(_.name == ut))
```

Comment:
maybe call `userGithubTeams` to explain what it is?
You may even want to consider passing `githubTeams` as a param to `cleanseUserTeams`?

Reply:
That's done now